### PR TITLE
Including cal.com domain

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Correcting domain in button

**Environment**: Staging(main branch) / Production

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: apps/web/pages/v2/settings/my-account/profile.tsx [hunk 1]
```diff
@@ -166,7 +166,7 @@ const ProfileView = () => {
               <TextField
                 name="username"
                 label={t("personal_cal_url")}
-                addOnLeading="https://"
+                addOnLeading="https://cal.com/"
```
Comment (zomars) [inline apps/web/pages/v2/settings/my-account/profile.tsx:169]: Let's use the actual hostname in a followup

**Review by zomars — APPROVED** [review]
LGTM

File: apps/web/pages/v2/settings/my-account/profile.tsx (hunk 1 above)
Comment (alannnc) [inline apps/web/pages/v2/settings/my-account/profile.tsx:169]:
```suggestion
                addOnLeading={process.env.NEXT_PUBLIC_WEBSITE_URL}
```

Comment (zomars) [reply apps/web/pages/v2/settings/my-account/profile.tsx:169]: Do all self-hosted users use `NEXT_PUBLIC_WEBSITE_URL`?

Comment (alannnc) [reply apps/web/pages/v2/settings/my-account/profile.tsx:169]: I believe after enviroments migrations this should be the one that everybody should use.

Comment (alannnc) [reply apps/web/pages/v2/settings/my-account/profile.tsx:169]: or NEXT_PUBLIC_WEBAPP_URL you mean?

**Pull request merged**